### PR TITLE
Allow custom commit message and pull request title

### PR DESCRIPTION
**Pull request by thehedgefrog**
This PR creates two new inputs, to allow customizing the PR title and commit message.  This is useful when someone wants to ensure consistency in their repo, such as using [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) or any organization standard.

Those are optional, and will default to the current value of 'Update GitHub Action Versions'.

New inputs were also documented in the README file.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -312,6 +312,10 @@ def print_message(message, message_type=None):
     email = os.environ['INPUT_COMMITTER_EMAIL']
     # Actions that should not be updated
     ignore = os.environ['INPUT_IGNORE']
+    # Commit message
+    commit_message = os.environ['INPUT_COMMIT_MESSAGE']
+    # Pull Request Title
+    pr_title = os.environ['INPUT_PULL_REQUEST_TITLE']
```
Comment (saadmk11) [inline main.py:318]: I think we should pass these to the `GitHubActionsVersionUpdater` class `__init__()` method, just like `ignore` input.

**Review by saadmk11 — CHANGES_REQUESTED** [review]
Hi @thehedgefrog, Thanks for The PR! :100:

File: main.py (hunk 1 above)
Comment (thehedgefrog) [reply main.py:318]: I am very much a newbie with Python, I will look into this and figure it out :)

Comment (thehedgefrog) [reply main.py:320]: Hi @saadmk11 - added it but I am not sure it was in the spirit of your suggestion.  Let me know and thank you!

File: main.py [hunk 2]
```diff
@@ -15,10 +15,12 @@ class GitHubActionsVersionUpdater:
     github_url = 'https://github.com/'
     action_label = 'uses'
 
-    def __init__(self, repository, base_branch, token, ignore_actions=None):
+    def __init__(self, repository, base_branch, token, commit_message, pr_title, ignore_actions=None):
         self.repository = repository
         self.base_branch = base_branch
         self.token = token
+        self.commit_message = commit_message
+        self.pr_title = pr_title
```
Comment (saadmk11) [inline main.py:23]:
```suggestion
        self.commit_message = commit_message or 'Update GitHub Action Versions'
        self.pr_title = pr_title or 'Update GitHub Action Versions'
```

File: main.py [hunk 3]
```diff
@@ -15,10 +15,12 @@ class GitHubActionsVersionUpdater:
     github_url = 'https://github.com/'
     action_label = 'uses'
 
-    def __init__(self, repository, base_branch, token, ignore_actions=None):
+    def __init__(self, repository, base_branch, token, commit_message, pr_title, ignore_actions=None):
```
Comment (saadmk11) [inline main.py:18]:
```suggestion
    def __init__(self, repository, base_branch, token, commit_message=None, pr_title=None, ignore_actions=None):
```

File: main.py [hunk 4]
```diff
@@ -326,7 +332,7 @@ def print_message(message, message_type=None):
 
     # Initialize GitHubActionsVersionUpdater
     actions_version_updater = GitHubActionsVersionUpdater(
-        repository, base_branch, token, ignore_actions=ignore
+        repository, base_branch, token, commit_message, pr_title, ignore_actions=ignore
```
Comment (saadmk11) [inline main.py:335]:
```suggestion
        repository, base_branch, token, commit_message=commit_message, pr_title=pr_title, ignore_actions=ignore
```

**Review by saadmk11 — COMMENTED** [review]
Hey @thehedgefrog, I have suggested a few small changes so that we can run the `GitHubActionsVersionUpdater` without needing the user inputs.

**Comment by thehedgefrog** [comment]
@saadmk11 thank you so much for this, it's very helpful for my understanding of Python!  Also, I caught how my testing was not complete.  Cheers!

File: main.py [hunk 5]
```diff
@@ -183,7 +185,7 @@ def create_new_branch(self):
         )
         subprocess.run(['git', 'add', '.'])
         subprocess.run(
-            ['git', 'commit', '-m', 'Update GitHub Action Versions']
+            ['git', 'commit', '-m', commit_message]
```
Comment (saadmk11) [inline main.py:188]:
```suggestion
            ['git', 'commit', '-m', self.commit_message]
```

File: main.py [hunk 6]
```diff
@@ -196,7 +198,7 @@ def create_pull_request(self, branch_name, body):
         """Create pull request on GitHub"""
         url = f'{self.github_api_url}/repos/{self.repository}/pulls'
         payload = {
-            'title': 'Update GitHub Action Versions',
+            'title': pr_title,
```
Comment (saadmk11) [inline main.py:201]:
```suggestion
            'title': self.pr_title,
```

**Review by saadmk11 — CHANGES_REQUESTED** [review]

**Review by saadmk11 — APPROVED** [review]
Thanks for addressing the change request. This looks great! :rocket:

**Pull request merged**